pages/1.9/release-notes/1.9.9/index.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.9.9
+title: Release Notes for 1.9.9
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.9.9
+---
+
+DC/OS 1.9.9 was released on July 10, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.9/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.9.9 includes the following:
+
+- Updated to Mesos 1.2.x ([change log](https://github.com/dcos/dcos/blob/1.9.9/packages/mesos/buildinfo.json)).
+- Updated to Marathon 1.4.11 ([change log](https://github.com/mesosphere/marathon/releases/tag/v1.4.11)).
+- Updated to Metronome 0.3.2 ([change log](https://github.com/dcos/metronome/releases/tag/v0.3.2)).
```

Comment:
This is wrong. **The Metronome version is 0.3.5**
 https://github.com/dcos/dcos/blob/1.9.9/packages/metronome/buildinfo.json